__tests__/buildTranslationFiles.spec.ts

```diff
@@ -238,6 +240,32 @@ describe.each(formats)('buildTranslationFiles in %s', (fileFormat) => {
       });
     });
 
+    describe('Self-closing', () => {
```

Comment:
I'd integrate these features into the existing tests, not sure we need a new suite for them.
WDYT?

Reply:
Done